org.gnuhealth.MyGNUHealth.yaml

```diff
@@ -0,0 +1,366 @@
+        sha256: a0af1213f3c2226497a97e2b3aa01a7e4bee4f403f95be16fc9acd2947514a78
+        x-checker-data:
+          name: webencodings
+          packagetype: bdist_wheel
+          type: pypi
+  # 'tinycss' is a direct dependency of mygnuhealth
+  # Generated with flatpak-pip-generator --runtime org.freedesktop.Sdk//23.08 --yaml --checker-data pip tinycss
+  - name: python3-tinycss
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "tinycss" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/05/59/af583fff6236c7d2f94f8175c40ce501dcefb8d1b42e4bb7a2622dff689e/tinycss-0.4.tar.gz
+        sha256: 12306fb50e5e9e7eaeef84b802ed877488ba80e35c672867f548c0924a76716e
+        x-checker-data:
+          name: tinycss
+          type: pypi
+
+  # Generated with flatpak-pip-generator --runtime org.freedesktop.Sdk//23.08 --yaml --checker-data pip cssselect
+  - name: python3-cssselect
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "cssselect" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/06/a9/2da08717a6862c48f1d61ef957a7bba171e7eefa6c0aa0ceb96a140c2a6b/cssselect-1.2.0-py2.py3-none-any.whl
+        sha256: da1885f0c10b60c03ed5eccbb6b68d6eff248d91976fcde348f395d54c9fd35e
+        x-checker-data:
+          name: cssselect
+          packagetype: bdist_wheel
+          type: pypi
+  # Runtime dependencies of Kivy
+  - name: libXmu
+    buildsystem: autotools
+    sources:
+    - type: archive
+      url: https://gitlab.freedesktop.org/xorg/lib/libxmu/-/archive/libXmu-1.1.2/libxmu-libXmu-1.1.2.tar.gz
+      sha256: c74dd369191d125e2b278e669301b7f1a0d67e2f6c92c7dad24e08a5e2cfcb45
+
+  - name: xclip
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/astrand/xclip/archive/0.13.tar.gz
+        sha256: ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758
+
+  - name: mtdev
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://bitmath.org/code/mtdev/mtdev-1.1.6.tar.gz
+        sha256: 1325f389a2f25cd5f5a8ea4d29aad24aa7c3ec30401d679400dd79eb9c0a8dbb
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} config-aux
+  - name: mygnuhealth
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 org.gnuhealth.mygnuhealth.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key="Icon" --set-value="${FLATPAK_ID}" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 org.gnuhealth.mygnuhealth.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
```

Comment:
you should really use the same app-id. (it's case sensitive)

Reply:
which is also why it can't find the icon.

Reply:
Where is it not using app-id? On the target side, there appears to be `${FLATPAK_ID}` everywhere to me, am I missing something?